wiki/osu!_Program_Files/User_configuration_file/fr.md

```diff
@@ -0,0 +1,230 @@
+
+***[Retour à la page principale](../ "Osu!_Program_Files")***
+
+![Combo Fire](/wiki/shared/combo-fire.jpg "Combo Fire") Ceci est une liste d'option située dans le fichier osu!.\<nomdevotrepc\>.cfg. Beaucoup de ces options son disponibles en jeu et il est recommandé d'y modifier ces options par son biais. N'éditez ces valeurs manuellement qu'en cas de vrai besoin.
```

Comment:
better example; like this:
```
![Combo Fire](/wiki/shared/combo-fire.jpg "Combo Fire")

Ceci est une 
```

otherwise the image is placed inline next to the text as a single (big) line